service/static/js/rest_api.js

```diff
@@ -271,4 +271,33 @@ $(function () {
 
     });
 
+    // ****************************************
+    // Toggle the status of a recommendation
+    // ****************************************
+
+    $("#toggle-btn").click(function () {
+
+        var product_id = $("#recommendation_product_id").val();
+        var related_product_id = $("#recommendation_related_product_id").val();
+
+        if (product_id && related_product_id){
+            var ajax = $.ajax({
+                type: "PUT",
+                url: "/recommendations/" + product_id + "/" + related_product_id + "/toggle"
+            })
+
+            ajax.done(function(res){
+                flash_message("Success")
```

Comment:
We need to update the form with the updated recommendation don't we?

Reply:
Not sure. Are you saying if after I used toggle button the form should refill with the new data? 
In this regard I think a better way would be we clear the form and list the new data in the bottom table. So it's more intuitive that an update or a toggle has done.

Reply:
Also it looks like the toggle endpoint only returns the status field on success. So either we need to call a get afterwards to get the whole data, or we probably need another story to modify the return object of toggle endpoint.

Reply:
Yeah sure we can have the table at the bottom updated as well, but there is no point in having out dated data in the form after toggle has been executed. Also, since this is a very minor I think it is okay to do it in the same story, but you are welcome to create another story.

Reply:
Okay I will clean it up.

Reply:
Hey @lj894, could you also please add $("#recommendation_status").val("") in line 23 clean form data function, this will make it possible to search all recommendations including false and true?